src/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -115,6 +115,66 @@ public void DirectoryLongerThanMaxLongPath_DoesntThrow()
             });
         }
 
+        [ConditionalFact(nameof(CanCreateSymbolicLinks))]
+        public void SymLinksMayExistIndependentlyOfTarget()
+        {
+            var path = GetTestFilePath();
+            var linkPath = GetTestFilePath();
+
+            Directory.CreateDirectory(path);
+            Assert.True(MountHelper.CreateSymbolicLink(linkPath, path, isDirectory: true));
+
+            // Both the symlink and the target exist
+            Assert.True(Directory.Exists(path), "path should exist");
+            Assert.True(Directory.Exists(linkPath), "linkPath should exist");
+            Assert.False(File.Exists(linkPath));
+
+            // Delete the target.  The symlink should still exist.  On Unix, the symlink will now be
+            // considered a file (since it's broken and we don't know what it'll eventually point to).
+            Directory.Delete(path);
+            Assert.False(Directory.Exists(path), "path should now not exist");
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Assert.True(Directory.Exists(linkPath), "linkPath should still exist as a directory");
+                Assert.False(File.Exists(linkPath), "linkPath should not be a file");
+            }
+            else
+            {
+                Assert.False(Directory.Exists(linkPath), "linkPath should no longer be a directory");
+                Assert.True(File.Exists(linkPath), "linkPath should now be a file");
+            }
+
+            // Now delete the symlink.
+            // On Unix, deleting the symlink should fail, because it's not a directory, it's a file.
+            // On Windows, it should succeed.
+            try
+            {
+                Directory.Delete(linkPath);
+                Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Windows), "Should only succeed on Windows");
+            }
+            catch (IOException)
+            {
+                Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Windows), "Should only fail on Unix");
+                File.Delete(linkPath);
```

Comment:
Everything will get cleaned up even without these deletes, thanks to the base class. These deletes are just to verify behavior.